component-samples/demo/README.MD

```diff
@@ -458,26 +458,80 @@ This implementation will trust Ownership vouchers with X5Chain encoding without
   + MySQL
-  + Postgres DB 
+  + PostgreSQL
 
-Make sure that `hibernate.cfg.xml` and `hibernate-properties` in `service.yml` is configured specifically for the selected backend.
 
-Sample `hibernate-properties` for MySQL Database
+1. Follow the below steps to configure FDO services with MySQL database: 
 
-```yaml
-hibernate-properties:
-  hibernate.connection.username: fdo
-  hibernate.connection.password: sample_password
-  hibernate.connection.driver_class: com.mysql.cj.jdbc.Driver
-  hibernate.connection.url: jdbc:mysql://host.docker.internal:3306/fdo?createDatabaseIfNotExist=true&useSSL=false&allowPublicKeyRetrieval=true
-  hibernate.dialect: org.hibernate.dialect.MySQLDialect
-  hibernate.dialect.storage_engine: innodb
-```
+   a. Make sure that `connection.driver_class` is commented out in `hibernte.cfg.xml`.
+
+   b. Update `hibernate-properties` in `service.yml`
+
+      ```yaml
+      hibernate-properties:
+        hibernate.connection.username: fdo
+        hibernate.connection.password: sample_password
+        hibernate.connection.driver_class: com.mysql.cj.jdbc.Driver
+        hibernate.connection.url: jdbc:mysql://host.docker.internal:3306/fdo?createDatabaseIfNotExist=true&useSSL=false&allowPublicKeyRetrieval=true
+        hibernate.dialect: org.hibernate.dialect.MySQLDialect
+        hibernate.dialect.storage_engine: innodb
+        hibernate.connection.requireSSL: false
+        hibernate.connection.autoReconnect: true
+      ```
+
+    c. Make sure that `useSSL` and `requireSSL` is set to `false` in `service.env` file.
+
+    d. Command to start MySQL server
+
+      ```shell
+      docker run -d -p 3306:3306 --name mysql-docker-container -e MYSQL_ROOT_PASSWORD=root-password -e MYSQL_DATABASE=fdo -e MYSQL_USER=sample-user -e MYSQL_PASSWORD=sample-password mysql/mysql-server:8.0.31
+      ```
+
+      For RHEL environment,
+
+      ```shell
+      sudo podman run -d -p 3306:3306 --name mysql-docker-container -e MYSQL_ROOT_PASSWORD=root-password -e MYSQL_DATABASE=fdo -e MYSQL_USER=sample-user -e MYSQL_PASSWORD=sample-password mysql/mysql-server:8.0.31
+      ```
+
+
+2. Follow the below steps to configure FDO services with PostgreSQL database: 
+
+   a. Make sure that `connection.driver_class` is commented out in `hibernte.cfg.xml`.
+
+   b. Update `hibernate-properties` in `service.yml`
+
+      ```yaml
+      hibernate-properties:
+        hibernate.connection.username: fdo
+        hibernate.connection.password: sample_password
+        connection.driver_class: org.postgresql.Driver
+        hibernate.connection.autoReconnect: true
+        hibernate.connection.requireSSL: false
+        hibernate.connection.url: jdbc:postgresql://host.docker.internal:5432/fdo?createDatabaseIfNotExist=true&useSSL=false
+        hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
+      ```
+
+
+    c. Make sure that `useSSL` and `requireSSL` is set to `false` in `service.env` file.
+
+    d. Command to start PostgreSQL server
```

Comment:
Use the below command to start PostgreSQL server

Reply:
Updated.